inshorts.py

```diff
@@ -0,0 +1,95 @@
+# inshorts module contains Inshorts class.
+# inshorts class will be used to fetch the news from inshorts.com as API response for it's instance.
+
+# standard library's
+import requests
+
+
+class Inshorts:
+    """
+    Inshorts module is the main module which will get data from Inshorts API
+    Inshorts class helps in providing 2 varities of News(all_news & Category_news).
+    """
+
+    url = "https://inshorts.com/api/en"
+
+    # def __init__(self,ob_name):
+    #     """Initialize Objects with it's name"""
+    #     self.ob_name = ob_name
+
+    def all_news(self, max_limits=5):
+        """Provides the all news via Inshorts API
+
+        Args:
+            max_limit: maximum limit of the selected news.
+        """
+
+        try:
+            url = f"{self.url}/news"
+            params = (
+                ("category", "all_news"),
+                ("max_limit", f"{max_limits}"),
+                ("include_card_data", "true"),
+            )
+            response = requests.get(url, params=params)
+            json_response = response.json()
```

Comment:
Please use the Response validation.
else how you will be sure that the response received is valid - it will lead to error in different stage of code
use response.raise_for_status method to validate status code.